examples/infra-linux-policyfile-cookbook/kitchen.yml

```diff
@@ -14,6 +14,12 @@ verifier:
 platforms:
   - name: ubuntu-16.04
 
+lifecycle:
+  pre_create:
+    # Export archive as a tar (removes need to install hab and pull/transfer keys)
+    # This is done in the studio since export as tar is only supported on Linux
+    - hab studio run 'cd results && source last_build.env && hab pkg export tar $pkg_artifact && mv $pkg_origin-$pkg_name-$pkg_version-$pkg_release.tar.gz hab_artifact.tar.gz'
```

Comment:
My main concern is if this is only supported on Linux what is the Windows way to do something similar. It is critical that we have both Windows and Linux remain as close to similar as physically possible.

Reply:
I'd at least like to see a solid path for the Windows side before we go ahead with this. Windows needs to be a primary operating system for testing or we are going to fall into the same trap as other projects and have a less than ideal Windows experience.

Reply:
@TheLunaticScripter I agree with this method of support, but I don't think we should block this PR because a Windows example for this workflow has not been created yet. @jerryaldrichiii is building this offline-driven example for Linux, and that's OK for now.

Once a Windows version of this example is created, then they will need to move in lockstep with each other.

Reply:
I'm good with that but let's be sure to open an issue for the windows example.